tests/packages/gen/prog/geningen.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This script generates input tests in parallel.
+# Test names are read from file "tests" in the same directory split by spaces.
```

Comment:
Before settling on a specific ingen shell script which we will use, I first want to ask around. Maybe someone has some ready scripts, or maybe someone has some ideas how the script can be improved. The decision for merging this PR is up to you, I just want to postpone the review of this specific file.

Reply:
Sure, I think we should leave it for testing if the command works